db/migrate/20150204075501_create_course_achievements.rb

```diff
@@ -0,0 +1,13 @@
+class CreateCourseAchievements < ActiveRecord::Migration
+  def change
+    create_table :course_achievements do |t|
+      t.references :course, null: false
+      t.string :title, null: false
+      t.text :description
+      t.integer :position, null: false
```

Comment:
I prefer something like 'weight', because 'position' implies it follows some natural sequence (like the integers)

but because of how databases are implemented, it might sometimes not be in running order, but it's the relative weights of the items that matter.